Confusing `opam source --dev-repo` message

It reports an error:

```
$ opam source --dev-repo opam-publish
[ERROR] /Users/thomas/git/opam-publish.0.2.1
[opam-publish] https://github.com/AltGr/opam-publish.git updated
```

But it worked fine.

```
$ opam config report
# OPAM status report
# opam-version    1.2.0 (1b004135bc57fc36f689a0b4c54b47fa761d6ce2)
# self-upgrade    no
# os              darwin
# external-solver aspcud
# criteria        -removed,-notuptodate,-changed*
# jobs            1
# repositories    1* (http), 1 (version-controlled)
# pinned          1 (version control)
# current-switch  4.02.1
# last-update     2014-12-16 00:26
```

--- a/src/client/opamArg.ml
+++ b/src/client/opamArg.ml
@@ -1669,7 +1669,6 @@ let source =
           | Some k -> k
           | None -> assert false
         in
-        OpamGlobals.error "%s" (Dir.to_string dir);
         mkdir dir;
         match
           OpamProcess.Job.run

